client/my-sites/domains/domain-management/dns/dns-records-list-item.jsx

```diff
@@ -16,12 +16,15 @@ function DnsRecordsListItem( { type, name, value, actions, disabled, isHeader, r
 				/>
 			}
 			popoverClassName="dns-records-list-item__action-menu-popover"
-			position="top left"
-			disabled={ actions.length === 0 }
+			position="bottom left"
 		>
 			{ actions.map( ( action ) => {
 				return (
-					<PopoverMenuItem key={ key++ } onClick={ () => action.callback( record ) }>
+					<PopoverMenuItem
+						disabled={ record.protected_field }
+						key={ key++ }
+						onClick={ () => ! record.protected_field && action.callback( record ) }
```

Comment:
Maybe checking `! record.protected_field` here is redundant because we're already checking it in the `disabled` attribute?

Reply:
That was just a double-check. I'll remove it 😄